src/main/java/com/gamingmesh/jobs/listeners/JobsPaymentListener.java

```diff
@@ -1634,7 +1633,11 @@ public void onPlayerInteract(PlayerInteractEvent event) {
 		- Jobs.getNms().getDurability(Jobs.getNms().getItemInMainHand(p)) != hand.getMaxDurability())
 		return;
 
-	    Bukkit.getScheduler().scheduleSyncDelayedTask(plugin, () -> Jobs.action(jPlayer, new BlockActionInfo(block, ActionType.STRIPLOGS), block), 1);
+		// either it's version 1.13+ and we're trying to strip a normal log like oak,
+		// or it's 1.16+ and we're trying to strip a fungi like warped stem
+		if ((Version.isCurrentEqualOrHigher(Version.v1_13_R1) && block.getType().toString().endsWith("_LOG")) ||
+		    (Version.isCurrentEqualOrHigher(Version.v1_16_R1) && block.getType().toString().endsWith("_STEM")))

```

Comment:
Also check for `_hyphae`

Reply:
Oops, I forgot that existed. I'll also add in a check for `_WOOD` so both `oak_wood` (all-sided bark) and `oak_log` will be counted.